fetch.bs

```diff
@@ -2689,9 +2738,29 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
      <a for=request>response tainting</a> to
      "<code>opaque</code>".
 
-     <li><p>Return the result of performing a <a>scheme fetch</a>
-     using <var>request</var>.
+     <li><p>Let <var>noCorsResponse</var> be the result of performing a <a>scheme fetch</a> using
+     <var>request</var>.
      <!-- file URLs end up here as they are not same-origin typically. -->
+
+     <li>
+      <p>If <var>noCorsResponse</var> is not a <a>filtered response</a> and the <a>CORB check</a>
+      with <var>request</var> and <var>noCorsResponse</var> returns <b>blocked</b>, then:
```

Comment:
I'd flip this around, and immediately return `noCorsResponse` if it isn't filtered or passes the CORB check.

Reply:
Maybe.

@annevk - can you please help me understand why you've added "If <var>noCorsResponse</var> is a <a>filtered response</a>" in c4a5a28?  Some filtered responses do not filter out the response body (e.g. a basic filtered response), so it seems to me that they should still be subject to CORB.  What am I missing? :-)

FWIW, I've removed the filtered-response wording and followed @jakearchibald's suggestion. 
 Please note that in the current Chromium implementation CORB is applied across the board (possibly on top, or rather before any filtered-response processing, which AFAIK is happening inside Chromium's renderer processes).  Please shout if you think that removing the filtered-response wording from the PR was a mistake.

Reply:
You're missing service workers. This can get a response that isn't filtered at all at which point poking at it would result in a null pointer exception or some such (if specifications had those).

Reply:
Okay - I think you're saying that populating corbSanitizedResponse might fail if noCorsResponse is a filtered response, because in this case noCorsResponse might not contain status / HTTPS state / CSP list.  Is that the concern?

I've tried to fix this by adding an extra step that I think should address your concern:
```
    If <var>noCorsResponse</var> is a <a>filtered response</a>, then return    
    <var>noCorsResponse</var>.
```